qiime2/core/transform.py

```diff
@@ -135,12 +135,12 @@ def coerce_view(self, view):
 
         return view
 
-    def validate(self, view, level='min'):
+    def validate(self, view, validate_level='min'):
```

Comment:
Grep has overshot the replacement I think.

The general rule here was:
`validate(..., level=...)`

anywhere else that calls validate as a side-effect (rather than express purpose):
 `unrelated(validate_level=...)`